### DISCUSSION TITLE
Environment variables are set in the 'Tests' of the request, but while using the flows ,these variable values are not getting fetched in the environment

### DISCUSSION THREAD

**Discussion by Debolinac** [postmanlabs/postman-flows#165] [Q&A]

**Comment by samay-v** [comment]
hey @Debolinac 
Let me know if this write up helps - https://github.com/postmanlabs/postman-flows/discussions/142

**Reply by bbbford** [reply]
They changed the functionality. Now environmental variables do not change the flow. No idea why they did this. Flows are now completely useless.
